cachi2/core/package_managers/npm.py

```diff
@@ -192,6 +199,19 @@ def fetch_npm_source(request: Request) -> RequestOutput:
         for dependency in info["dependencies"]:
             components.append(Component.from_package_dict(dependency))
 
+    npm_deps_dir = request.output_dir.join_within_root("deps", "npm")
+    npm_deps_dir.path.mkdir(parents=True, exist_ok=True)
+
+    # Download dependencies via resolved URLs
+
+    results = asyncio.run(
+        get_dependencies(
+            npm_deps_dir,
+            info["dependencies_to_download"],
+            5
+        )
+    )
```

Comment:
We're missing checksum verification

Reply:
I suppose for checksum verification we want to use same methods from https://github.com/containerbuildsystem/cachi2/blob/main/cachi2/core/checksum.py#L27 , right?